tests/test_typescript_features.py

```
@@ -0,0 +1,524 @@
+        result = run_tldr(["structure", tmpdir])
+
+        assert "files" in result
+        file_info = result["files"][0]
+
+        classes = file_info.get("classes", [])
+        assert any("Calculator" in c for c in classes), \
+            f"should find 'Calculator' class, got: {classes}"
+
+    def test_structure_finds_methods(self, temp_ts_file):
+        """structure should detect class methods."""
+        filepath, tmpdir = temp_ts_file(CLASS_WITH_METHODS)
```

Ruff (0.14.11)

Unpacked variable `filepath` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)